code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -211,6 +225,11 @@
 
 
 /obj/structure/machinery/computer/shuttle/dropship/flight/attack_alien(mob/living/carbon/xenomorph/xeno)
+	// if the shuttleid is null or the shuttleid references a shuttle that has been removed from play, pick one
+	if(!shuttleId || !SSshuttle.hasShuttle(shuttleId))
```

Comment:
As far as I can tell, hijack can only occur for the primary LZ, and the primary LZ is already going to be assigned to the alamo; so hijack will still occur as it has.

I also tried sending the alamo to infinite transit and that just causes a different runtime: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/77514c13-5e38-4ee7-b4bb-8168c456b6c4)

Reply:
I removed the overrides in the lz1 and lz2 subclasses of the flight computer which would hard code it in. I'll work on the runtimes - thanks for the testing